docs/README.md

```diff
@@ -38,181 +38,92 @@ Code | Text | Description
 
 ### Headers and request signing
 
-All API calls need to be signed using HMAC and SHA256 or SHA512.
+All API calls need to be signed using HMAC and SHA-256 or SHA-512. When a request contains a body, the body must be valid JSON and `content-type` header with value `application/json; charset=utf-8` should be included. Other encodings may work too, but using UTF-8 is preferred.
 
-HMAC is calculated from the HTTP request's body payload using the signing secret. Header keys must be sorted alphabetically, lower-cased and included in HMAC calculation.
+All API responses are signed the same way allowing merchant to verify response validity. In addition the reponses contain `cof-request-id` header. Saving or logging the value of this header is recommended.
```

Comment:
All API responses are signed the same way allowing merchant to verify response validity. In addition the reponses contain `cof-request-id` header. -> All API responses are signed the same **way,** allowing merchant to verify response validity. In **addition,** the reponses contain `cof-request-id` header.